src/dao/pages/CreateADAO/forms/DAODetailsForm.tsx

```diff
@@ -95,6 +95,21 @@ export function DAODetailsForm() {
           fallback={<DefaultLogoIcon boxSize="4rem" alignSelf="end" color={Color.Grey_Blue._100} />}
         />
       </Flex>
+      <FormInput<"infoUrl">
```

Comment:
did we confirm the design to where add `Info Url` in UI , since we do not have `Info URL` for multisig

Reply:
Done.